packages/jellyfish-api-core/__tests__/category/account/removePoolLiquidity.test.ts

```diff
@@ -0,0 +1,112 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+
+describe('Poolpair', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+  let poolLiquidityAddress: string
```

Comment:
yea we will not use global variable in test to keep test independence
so you might need declare `poolLiquidityAddress` in each test

Reply:
Okay, do you mean in the other tests within this test script that `poolLiquidityAddress` should be declared? or generally in other tests scenarios should`poolLiquidityAddress` be required? 

<img width="577" alt="Screenshot 2021-06-26 at 12 28 11 AM" src="https://user-images.githubusercontent.com/65014479/123456435-690c4080-d615-11eb-9bac-f94b77b118f3.png">

I tried adding `let poolLiquidityAddress: string` within other tests sections but it throws an error `TS2339: Property 'length' does not exist on type 'Promise<string>'.`, and line 9's `let poolLiquidityAddress: string` becomes greyed out with error `Unused local variable poolLiquidityAddress`

Reply:
yea it stated `ln: 9: let poolLiquidityAddress: string is unused`, that can be removed ady

Reply:
Understand, my bad it's called in line 77 as a `params`. Will leave it for now.